packages/store/src/patterns/exo-makers.js

```diff
@@ -1,8 +1,18 @@
+/* global globalThis */
 import { objectMap } from '@agoric/internal';
 
 import { defendPrototype, defendPrototypeKit } from './exo-tools.js';
 
-const { seal, freeze } = Object;
+const { seal, freeze, defineProperty } = Object;
+
+// TODO Use environment-options.js currently in ses/src after factoring it out
+// to a new package.
+const env = (globalThis.process || {}).env || {};
+
+// Turn on to give each exo instance its own toStringTag value.
+const LABEL_INSTANCES = (env.DEBUG || '')
```

Comment:
`$DEBUG` is a pretty short name, if this is really coming from a Unix process environment, it's like to be enabled by lots of unrelated things. Maybe `EXO_DEBUG` or something slightly more distinctive? But, I haven't been in a position to use this, and a longer name could be an ergonomic problem, if you think it's the right one to use, I'll go with it. And looking for `label-instances` within it is certainly less likely to be randomly triggered.

Reply:
Yeah, I'm going to leave this conversation unresolved until we discuss. We need stronger conventions around naming such environment-option configuration variable names. I saw the `DEBUG` variable name was already in use, and was already "parsed" (split on `:`) to detect individual debug settings. OTOH, `lockdown` uses one variable name per configurable option, with the conceptual grouping done by common prefix. 

Do we already have a ticket around communicating such configuration options?

Reply:
Found the ticket: https://github.com/Agoric/agoric-sdk/issues/3581 . Updated to point at this conversation.

The merged https://github.com/endojs/endo/pull/1554 and https://github.com/Agoric/agoric-sdk/pull/7409 are where the choice of `DEBUG` for this was made. Of course, we can revisit that. But this PR merely uses the convention that we already made in those other two PRs. These were only merged today, so perhaps I'm standing on ceremony too much on this. Nevertheless, changing that shouldn't hold up this PR; it should be distinct PRs. So Resolving.